remotespark/livyclientlib/livyclient.py

```diff
@@ -10,17 +10,18 @@ class LivyClient(object):
 
     def __init__(self, session):
         self._session = session
+        self._session.create_sql_context()
```

Comment:
Fair enough. To reduce the chance of overwriting, we might want to give the default sqlContext a bit more obscure of a name.